Source/WTF/wtf/PlatformHave.h

```diff
@@ -395,7 +395,7 @@
 #define HAVE_AVCONTENTKEYSESSION 1
 #endif
 
-#if (PLATFORM(MAC) && __MAC_OS_X_VERSION_MAX_ALLOWED >= 101504) \
+#if PLATFORM(MAC) \
     || (((PLATFORM(IOS) && !PLATFORM(IOS_FAMILY_SIMULATOR)) || PLATFORM(MACCATALYST)) && __IPHONE_OS_VERSION_MAX_ALLOWED >= 130400) \
     || (PLATFORM(WATCHOS) && !PLATFORM(IOS_FAMILY_SIMULATOR) && __WATCH_OS_VERSION_MAX_ALLOWED >= 60400) \
     || (PLATFORM(APPLETV) && !PLATFORM(IOS_FAMILY_SIMULATOR) && __TV_OS_VERSION_MAX_ALLOWED >= 130400) \
```

Comment:
Here even more than elsewhere, splitting the work into four PRs per platform seems counter-productive.

Reply:
Really? The PR I made originally was huge and had an error somewhere in it and I couldn’t figure out where. I think splitting it up is the only way for it actually to be reviewable and fully understandable.

Reply:
If those PRs were independent (so we could revert them one by one), it would make sense. Ditto if they are really truly big. But this one is tiny, and you'd be touching the same condition four times instead of one - that's a lot of work for everyone involved, and only harder to figure out what broke it if anything goes wrong. They may not even revert individually.

Reply:
Okay, I think what I'll do is do it year-by-year instead of year-by-year and platform-by-platform. That should make them independent and fewer in total.

Reply:
(I'll start this new plan after landing the current batch of patches I posted - no sense rewriting what already exists)